### ISSUE TITLE
secret key pop up - legitimate?

### ISSUE THREAD

**Issue by debrb22** [Fugiman/google-meet-grid-view#311]
When I opened Google Meet a pop-up appeared that asked me to enter a secret key if I wanted to be able to continue to use Grid View. I've been using Grid View for awhile and have never had this popup before. I have attached the first popup and then after I selected cancel the second popup appeared.

Is this popup real or is it a scam? Do I need to enter a secret key to continue using Grid View?
![popup1](https://user-images.githubusercontent.com/71326128/93210272-4ddadf00-f72d-11ea-876f-84f0a5ce43f3.jpeg)
![popup2](https://user-images.githubusercontent.com/71326128/93210279-516e6600-f72d-11ea-9491-1938f5462fd0.jpeg)

**Comment by capowitt** [comment]
Two red flags. "Turn of this alert" instead of "Turn off this alert" and "remove this extension the alert will disappear" instead of "remove this extension and the alert will disappear" or something similar. Even the word disappear does not sound like a speaker of English as a primary language. I've had the same alert. Malwarebytes did not find a problem, however.

**Comment by capowitt** [comment]
I believe now that the message is being generated by the "Grid View for Google Meet" extension for Google. I'm removing it.

**Comment by debrb22** [comment]
Yes, I think you're right. I uninstalled this extension and installed Google Meet Grid View instead.
